Marlin/configuration_store.cpp

```diff
@@ -1098,7 +1098,7 @@ void Config_ResetDefault() {
     #endif
     SERIAL_EOL;
     #if ENABLED(DISTINCT_E_FACTORS)
-      for (uint8_t i = 0; i < E_STEPPERS; i++) {
+      for (uint8_t i = 0; i < EXTRUDERS; i++) {
```

Comment:
is there another case, when except switching extruder Number of E_STEPPERS is not equal to number of EXTRUDERS ? 

I think MIXING-Extruder, but do they use EXTRUDER Parameter  ?